src/python/bcc/__init__.py

```diff
@@ -751,9 +751,17 @@ def get_kprobe_functions(event_re):
 
     def _check_probe_quota(self, num_new_probes):
         global _num_open_probes
-        if _num_open_probes + num_new_probes > _probe_limit:
+        if _num_open_probes + num_new_probes > BPF.get_probe_limit():
             raise Exception("Number of open probes would exceed global quota")
 
+    @staticmethod
+    def get_probe_limit():
+        env_probe_limit = os.environ.get('PROBE_LIMIT')
```

Comment:
One more request, could you change env name from PROBE_LIMIT to BCC_PROBE_LIMIT? This way, we have less possible conflict with other environ variables.

Reply:
update